### DISCUSSION TITLE
Github Pages - Custom domain Issue i cannot resolve.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#42760] [New to GitHub]
Dear Github Community,
i was watching some YouTube, when i figured. "Why not make my own Website. Just a small one, with some Buttons to promote myself".
Now after 24~ Hours, i still cannot figure out how to solve the issue i have.

![image](https://user-images.githubusercontent.com/121500487/209768474-d9c1508e-a1bf-4c63-be37-75b492dd5023.png)
> ^Picture showing settings/pages^

![image](https://user-images.githubusercontent.com/121500487/209768530-a8666918-28a1-454f-940f-37c801ad5963.png)
> ^Picture showing MyProject/settings/pages^

![image](https://user-images.githubusercontent.com/121500487/209768696-3a2c6bc7-e672-44d5-8658-3a74abdd3858.png)
> ^Added to my DNS settings^
> I also added the CNAME inside my repositorie.

Now i'm very consufed over here. There is alot more going on in my DNS settings on where i bought my domain. 
Do i have to remove them? Did i make a mistake somewhere? 
I would be very grateful if I could get help.

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/121500487/209773644-73effc33-696b-4e11-9ab5-4eb48e455d65.png)
![image](https://user-images.githubusercontent.com/121500487/209773656-15787f59-a4ee-4905-8b5f-185f3209912c.png)

I prob had to wait a little longer ... But now there is this issue. /defaultsite ? Where does that come from? I looked up my Github, there is nothing.

**Reply by denisbanach** [reply]
Looks like you have the same provider as me. Were you able to solve the problem? I have the same, I am also redirected to defaultsite.

**Comment by radar-blip** [comment]
Seems to be working fine now, this is what i'm seeing.
Maybe it just took some time.

![Screenshot from 2022-12-28 03-12-52](https://user-images.githubusercontent.com/39196767/209780853-5b57f1af-3a90-4e23-a5d3-bf85c5e1a9e2.png)

**Reply by ghost** [reply]
Yes, for some reason the "privacy.html" inside my repo was causing the issues. I have no idea why... After removing it, "defaultsite" was gone. Now that i have removed almost everything. My Original Website should be shown in a Minute. 

Thank you for the reply tho!!

**Reply by radar-blip** [reply]
No problem.
